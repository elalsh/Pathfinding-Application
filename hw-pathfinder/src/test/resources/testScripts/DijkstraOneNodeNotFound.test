# Find a path with Dijkstra's algorithm when
# one node isn't in the graph.

CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B

AddEdge simpleGraph A B 10.0
AddEdge simpleGraph A B 1.0

FindPath simpleGraph C B