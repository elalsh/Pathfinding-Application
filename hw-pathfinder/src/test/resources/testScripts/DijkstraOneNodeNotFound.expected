# Find a path with Dijkstra's algorithm when
# one node isn't in the graph.

created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph

added edge 10.000 from A to B in simpleGraph
added edge 1.000 from A to B in simpleGraph

unknown: C